CONTRIBUTING.md

```diff
@@ -309,11 +309,15 @@ For more about creating challenges, see [seed/README](seed/README.md) and [seed/
 
 #### Changes to the seed files
 
-If you made changes to any file in the `/seed` directory, you need to run
+If you made changes to any file in the `/seed` directory, you then need to stop the server by typing CTRL-C, then you need to run
```

Comment:
Actually you can fire up a fresh terminal alongside and run `node seed`. The server need not be stopped. Then you can simply type in `rs` anytime on the running server terminal to restart it on the fly without having to kill the process using CTRL+C.

Reply:
Ah alright, well at least one of the methods (the one I wrote, or the one you are describing if it is better) should be included in the instructions, because currently they are not complete enough for someone new to contributing.